sdk/FEATURES.md

```diff
@@ -166,7 +166,7 @@ Output parameters are a dictionary of string files that users can define as a co
 ### Input Artifacts
 
 Input Artifacts in Kubeflow pipelines are used for passing raw text or local files as files placed in the component pod. Since Input Artifacts can only be raw or in a compressed format as strings, we created a
-[custom step](https://github.com/kubeflow/kfp-tekton/blob/master/sdk/python/kfp_tekton/compiler/_op_to_template.py#L435) for passing these strings as files before the main task is executed. The [input_artifact_raw_value](/sdk/python/tests/compiler/testdata/input_artifact_raw_value.py)
+[custom step](/sdk/python/kfp_tekton/compiler/_op_to_template.py#L435) for passing these strings as files before the main task is executed. The [input_artifact_raw_value](/sdk/python/tests/compiler/testdata/input_artifact_raw_value.py)
```

Comment:
@Tomcli @fenglixa -- `_op_to_template.py#L435` seems to no longer refer to the correct code snippet.

Is this still a valid paragraph, what should be the corrected code snippet?

Reply:
that still applies since Tekton doesn't have native artifact support in v1beta1. Let's use the perm link here so the code won't be out of place.
https://github.com/kubeflow/kfp-tekton/blob/f6e61444a566b257ea72434f8f7aeb0e1002280f/sdk/python/kfp_tekton/compiler/_op_to_template.py#L427-L429

Reply:
I updated the link.